GitPrompt.ps1

```diff
@@ -133,7 +133,7 @@ function Write-GitStatus($status) {
         Write-Prompt $s.AfterText -BackgroundColor $s.AfterBackgroundColor -ForegroundColor $s.AfterForegroundColor
 
         if ($WindowTitleSupported -and $s.EnableWindowTitle) {
-            if( -not $Global:PreviousWindowTitle ) {
+            if (-not (Test-Path -Path Variable:Global:PreviousWindowTitle)) {

```

Comment:
Can you apply this strict revision to `elseif ( $Global:PreviousWindowTitle )` below? Getting this:

> The variable '$Global:PreviousWindowTitle' cannot be retrieved because it has not been set.